### DISCUSSION TITLE
Communicate using Markdown submission issue

### DISCUSSION THREAD

**Discussion by aliyabatoolalvi** [community/community#124635] [GitHub Education]
### Hi everyone,

my assignment got rejected by education of Communicate using Markdown (We were unable to find a Classroom assignment for your account.)

**Comment by jabulilemasilela** [comment]
I'm having the same issue, did you manage to fix it? If yes, please share

**Reply by aliyabatoolalvi** [reply]
hi, i couldn't that's why i started the discussion! 
I would appreciate if you share when you know.
